generate_homepage_from_content.py

```diff
@@ -27,7 +29,8 @@
 # HTML templates
 CONTENT_CONTAINER = '<div class="content %s">%s</div>'
 WEBSITE_CONTENT_TEMPLATE = ('<span class="website-content">%s</span>\n'
-                            '<span class="website-url">%s</span>')
+                            '<span class="website-url">'
+                            '<a href="%s">%s</a></span><br><br>')
```

Comment:
This `span` element may have been changed to `div` in @jillh510's latest diff, so maybe a moot point...? But regardless, I'd say, don't use `<br>` and accomplish the spacing with css styling instead.